app/logic/events.py

```diff
@@ -350,24 +352,28 @@ def validateNewEventData(data):
 
     if data['timeEnd'] <= data['timeStart']:
         return (False, "Event end time must be after start time.")
-
+    
     # Validation if we are inserting a new event
     if 'id' not in data:
 
-        event = (Event.select()
-                      .where((Event.name == data['name']) &
-                             (Event.location == data['location']) &
-                             (Event.startDate == data['startDate']) &
-                             (Event.timeStart == data['timeStart'])))
+        sameEventList = list((Event.select().where((Event.name == data['name']) &
+                                                   (Event.location == data['location']) &
+                                                   (Event.startDate == data['startDate']) &
+                                                   (Event.timeStart == data['timeStart'])).execute()))
+        
+        copyList = sameEventList.copy()
```

Comment:
I think this variable could be named to be more descriptive.

Reply:
Changed variable name to sameEventListCopy.